CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.3.0 (2019-02-06)
+### Added
+* Add support for Azure IoT distributed tracing

```

Comment:
Suggest adding suffix (preview) and the link to https://aka.ms/iottracing

Reply:
Added in PR: https://github.com/Microsoft/vscode-azure-iot-toolkit/pull/256/files